docs/plugins/lamp-plugin.md

```diff
@@ -177,6 +177,18 @@ steps:
 ### `apacheMods` {array}
 An array of Apache modules to enable (should be installed via [`installPackages`](#installpackages-array) if needed).
 {% endoption %}
+{% details Example %}
+{% highlight yaml%}
+- name: Install Drupal 7 with LDAP Apache Module installed.
```

Comment:
I was trying to keep the examples very specific to the option. So here I would make an example like:

```
steps:
  - name: Install Apache Mods
    plugin: LAMPApp
    apacheMods:
        - ldap
```
Get what I mean? The contextual examples can occur at the bottom of the page.